src/buildwin.rst

```diff
@@ -0,0 +1,32 @@
+Building on Windows
+==========================
+This document presents conda-forge and conda-build information and examples
+when building on Windows.
+
+Simple CMake-Based ``bld.bat``
+------------------------------
+Some projects provide hooks for CMake to build the project. The following
+example ``bld.bat`` file demonstrates how to build a traditional, out-of-core
+build for such projects.
+
+**CMake-based bld.bat:**
+
+.. code-block:: bat
+
+    setlocal EnableDelayedExpansion
```

Comment:
I don't think it matters much, but my inclination is to have echo on, like with bash and `set -x`